sources/web/datalab/polymer/components/datalab-files/datalab-files.html

```diff
@@ -49,6 +49,10 @@
         width: 20px;
         padding-right: 5px;
       }
+      .toolbar-button[disabled] {
+        color: var(--disabled-fg-color);
+        background-color: var(--secondary-bg-color);
```

Comment:
Are you really changing the background color when you disable a button? It seems like that should not be the case, and therefore this line should not be here.